shell/app-shell/elements/cloud-data.js

```diff
@@ -36,13 +38,12 @@ const template = html`
     on-user="_onForward"
   ></fb-user>
 
-  <fb-user-context
-    config="{{config}}"
-    userid="{{userid}}"
+  <user-geolocation on-coords="_onCoords"></user-geolocation>
+  <user-context
     context="{{context}}"
-    arc="{{arc}}"
-    on-friends="_onForward"
-  ></fb-user-context>
+    userid="{{userid}}"
+    coords="{{coords}}"
```

Comment:
Just to make sure you intended it, this PR looks like an omnibus one that also incorporates rework of user coordinates and some description fixes.

Reply:
There are some nuances. I had to do some modifications on User (and Geolocation) in order to make Restaurant demo function under this PR, I didn't want to leave it broken.